sys/include/usb/dfu.h

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (C) 2020 Dylan Laduranty
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    usb_dfu   DFU - USB Device Firmware Upgrade
+ * @ingroup     usb
+ * @brief       Generic USB DFU defines and helpers
+ *
+ * @{
+ *
+ * @file
+ * @brief       Definition for USB DFU interfaces
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef USB_DFU_H
+#define USB_DFU_H
+
+#include <stdint.h>
+
+#include "usb.h"
+#include "usb/descriptor.h"
+#include "usb/usbus.h"
+#include "usb/usbus/control.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define USB_IF_DESCRIPTOR_DFU           0x21   /**< USB DFU type descriptor */
+#define USB_DFU_VERSION_BCD             0x0110 /**< USB DFU version in BCD */
+
+/**
+ * @name    Default USB detach timeout for DFU descriptor
+ * @{
+ */
+#ifndef USB_DFU_DETACH_TIMEOUT
+#define USB_DFU_DETACH_TIMEOUT 255
```

Comment:
What unit is that in?
Better also change the define to include that, so e.g. `USB_DFU_DETACH_TIMEOUT_MS`